src/components/Button.test.js

```diff
@@ -0,0 +1,11 @@
+import renderer from 'react-test-renderer';
+import { Button } from "./Button";
+import theme from '../theme';
+import { ThemeProvider } from 'styled-components';
+
+describe("<Button/>", () => {
+  it("renders Button component correctly", () => {
+    const button = renderer.create(<ThemeProvider theme={theme}><Button>Button</Button></ThemeProvider>).toJSON();
```

Comment:
Here's a suggestion of a way to avoid having to import the ThemeProvider in every test: https://testing-library.com/docs/react-testing-library/setup#custom-render